src/createServer.js

```diff
@@ -1,3 +1,47 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+const { checkUrl } = require('./checkUrl');
+const { getParams } = require('./getParams');
+
+const createServer = () => {
+  const server = http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'application/json');
+
+    const [text, query] = req.url.split('?');
+    const { originalText, targetCase } = getParams(text, query);
+
+    const errors = checkUrl(originalText, targetCase);
+
+    if (errors.length) {
+      res.statusCode = 404;
```

Comment:
It's the wrong code. 404 - means that such endpoint wasn't found or some entity in the database, or requested case if talking closer to this example. All your errors are related to incorrect params that someone has sent you. So you must return statusCode: 400 for a Bad Request.